### ISSUE TITLE
As a user of linked data, I want the Sound and Vision schema to be aligned with schema.org

### ISSUE THREAD

**Issue by mwigham** [beeldengeluid/beng-lod-server#38]
In this issue we will map the Sound and Vision schema (that describes the data model of the archive, NOT the GTAA that describes metadata values) to schema.org. 

- [x] Review the iMMIx->schema.org mapping that Spinque created for data import to SpinqueDesk
- [x] Create a mapping for our new Sound and Vision RDF schema, for metadata fields that can be represented in schema.org (other metadata fields will be ignored)
- [x] Create some data examples of how the mapped data would appear for programmes
- [x] Document issues arising
- [x] Review with Willem

## Acceptance criteria
- [x] The mapping at least covers title, description, broadcast date , people, place and subject fields, plus the relationships between series, season, episode and clip.  
- [ ] The issues are clear enough to help us to formulate new stories for the next steps

**Comment by mwigham** [comment]
Issues are discussed in [this document](https://docs.google.com/document/d/1ujma9R8kq5X9M1pg-o-Hl8cv9rvtpHSwl3sFZ2ONPRU/edit?usp=sharing), which also contains links to the mapping.

**Comment by jblom** [comment]
@wmelder please give some smart sounding comments and then @mwigham can close it

**Comment by jblom** [comment]
@wmelder please review this story, so we can finally close it (grooming now)

**Comment by wmelder** [comment]
I could spend a lot more time at this review, but I won't. Here are some remarks:
- Most mappings seem accurate, but some still raise choice questions.
- I would really much appreciate it if the ordering of fields was more prioritized (like the creative works first, or whatever). The acceptance criteria contains a minimal requirement. Why not list this first and then continue with the rest? This would also make the review process more clear.
- For review of some of the fields I just don't have enough information (where is that immix manual when you need it). 
- We still need the mapping from our rdfs schema to schema.org.
- We could benefit from the fact that some fields are already mapped in our beng-lod server and share a target schema element. This might be another possibility to group fields, so that we have a better overview. I am not sure if this would imply a new column, but I guess it does and that it would immediately include the mapping from the rdfs to schema.org.

**Comment by mwigham** [comment]
@wmelder Thanks very much for the review. Could you please indicate which of your points I need to tackle in this issue, and which should be for future issues?

**Comment by wmelder** [comment]
@mwigham 
For making this document better readable and usable my suggestion is to:
- [ ] Add a column 'NISV schema' that can contain the element URI values. 
- [ ] For each element from the 'immix field' column add the target class or property (URI values). - [ ] When values are the same, group the rows, merge the cells from the 'nisv schema' column and merge the cels in the 'schema.org field' so that mapping only has to be done once.

The questions concerning the content of either the immix fields or the schema.org fields can be addressed later, probably together with someone like Tim, Vincent, Marjolein or Kim. 

When you think this issue is really about something else, then the above should be added to a new issue. Does this answer your question?

**Comment by mwigham** [comment]
@wmelder Thanks, I'll address those points and then create new issue(s) for the rest.

**Comment by mwigham** [comment]
@wmelder I've reorganised the spreadsheet, can you look and see if it is better now please? I decided not to merge the cells as that messes things up if we want to sort on different values, this way the spreadsheet stays flexible.

**Comment by jblom** [comment]
@wmelder yeah would be great to wrap this one up, so we can also get rid of it on the board

**Comment by wmelder** [comment]
@mwigham The document, spreadsheet, rdfs file are looking very good. It is hard to 'finish' reviewing, but for now I think it was thorough enough. We have a spreadsheet that lists the mappings for the main fields needed for linked data to schema.org. These mappings are already manifested in a Turtle document containg the RDFS subClassOf and subPropertyOf definitions. While we were working on this the scope was broadening with the DAAN data, that is completely different. Luckily, our nisv schema didn't change that much, so that was already pretty good describing the NISV domain. While working with the actual data on the LOD server, we noticed that it would be better not to load all the carrier information, since this is not so informative for discovery. So, therefor these fields now don't really have to be mapped. We discussed the status and decided to close the issue. The branch needs to be maintained. It cannot yet be merged, but does contains valueable RDFS definitions for future work.
